src/Components/Search/Suggestions/index.tsx

```diff
@@ -10,19 +12,60 @@ interface Props {
   viewer: SuggestionsSearch_viewer
 }
 
-export const SearchSuggestions: React.SFC<Props> = ({ viewer }) => {
-  const edges = get(viewer, v => v.search.edges)
-  if (edges.length > 0) {
-    return (
-      <>
-        <h1>Suggestions</h1>
-        {edges.map(({ node }) => {
-          return <div>{node.displayLabel}</div>
-        })}
-      </>
-    )
+interface State {
+  selectedType: string
+  selectedID: string
+}
+
+export class SearchSuggestions extends React.Component<Props, State> {
+  state = {
+    selectedType: null,
+    selectedID: null,
+  }
+
+  renderPreview() {
+    const { selectedID, selectedType } = this.state
+    if (selectedID && selectedType) {
+      return <SearchPreview entityID={selectedID} entityType={selectedType} />
+    }
+  }
+
+  render() {
+    const { viewer } = this.props
+    const edges = get(viewer, v => v.search.edges)
+    if (edges.length > 0) {
+      return (
+        <>
+          <Flex flexDirection={["column", "row"]}>
+            <Box width={["100%", "25%"]}>
+              <Flex flexDirection="column">
+                {edges.map(
+                  ({ node: { displayLabel, searchableType, href } }) => {
+                    return (
+                      <Box
+                        onClick={() => {
+                          // TODO: Expose `id` from `SearchableItem`
```

Comment:
cc @dblandin 

We'll need the id of items returned in autosuggest (so the user can click/select them and get the previews hooked up). For now I'm just requesting `href` and doing a regex replacement, but we should augment `SearchableItem` interface with id's (or maybe it should implement `NodeInterface`?).

Reply:
It looks like you took care of the metaphysics end [here](https://github.com/artsy/metaphysics/pull/1494). Do you want to update the code here to use that before we merge, or do you plan on making those changes in a follow-up PR?

Reply:
Oh, I'll do that in a followup. I _might_ need to tweak the MP schema a bit more.